devel/mps_load_test.sh

```diff
@@ -208,7 +202,8 @@ cd "$src/devel"
 
 # Build a version of mod_pagespeed with all optimizations enabled, but with
 # a build that includes DCHECKs.
-make -j8 CONF=$compile_mode apache_trace_stress_test_server \
```

Comment:
Ah, CONF=OptDebug overrides BUILDTYPE to equal Release in devel/Makefile. Let me debug this some more.